storage/index.js

```diff
@@ -1,7 +1,45 @@
+var createStorageBackend = require('./backend')
```

Comment:
Moved the `./backend` seam from `app.js` to `storage/index.js`. This is where we use `browserify` with `-r` to point to different implementations on different platforms, e.g. the browser and webxdc.